awesomerc.lua

```diff
@@ -372,6 +372,17 @@ awful.keyboard.append_global_keybindings({
                 end
             end
         end,
+    },
+    awful.key {
+        modifiers   = { modkey },
+        keygroup    = "numpad",
+        description = "select layout directly",
+        group       = "layout",
+        on_press    = function (index)
+            local keys_to_indices = "1234567890./*-+\n"
```

Comment:
I know I am being super conservative with what gets added to `rc.lua`, sorry about that.

This is *much* nicer than the previous attempt, but I think we can still hide a bit more of the imperative code away from `rc.lua` by adding a `numpad_indexed` group containing this.

Reply:
If you want me to. But I'm not so sure about that idea.

Users may decide that "obviously" 0 should go before 1, or Enter should go after 9. Having `keys_to_indices` fully exposed will allow them to customize it.

Reply:
Then maybe a category to only add 1..9+0 where 0==10, or just = 1..9? I am pretty much against adding imperative logic in `rc.lua`. It should stay/become a config file without imperative business logic. I like the feature, but we need some sort of compromise.

Adding the key groups was a compromise in the first place. The rational was the previous `rc.lua` had an imperative for-loop. I wanted to get rid of it for the same reason why I am asking for this change. However, as proven by this PR, the key groups are a leaky abstraction. Some people will inevitably want to add a limitless number of them for all weird keyboard layouts in the universe. For example, I own a Logitech G15-2nd-gen, G510 and G910. All of them have extra key groups for random tasks. Most mid-range external keyboards also have multimedia keys, but not the same ones and not the same key order. The problem with key groups is just that: there is no right way. In this case our problem is how to handle `/*-+`. Some people might argue you put them in the wrong order too. You see, this is tricky and given this is the default config, we have to please the largest subset of users. So maybe 1..9 or 1..9+10 are the most logic choice? I am looking for a way forward here.

Reply:
Okay, I understand your concerns here.

This commit depended on the definition of `numpad`, so I removed it, and pushed a new commit with your 1..9+10 proposal (8aa8c6c9dd413e5e7af42524284c46879acfb191). Is that okay?